kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/marketplace/presenter/MarketplacePresenter.java

```diff
@@ -245,6 +247,10 @@ public void onItemClicked(int position, OfferTypeEnum offerType) {
             }
         } else {
             offer = spendList.get(position);
+            if(offer.getContentType() == ContentTypeEnum.EXTERNAL) {
+                nativeSpendOfferClicked(offer);
```

Comment:
Does marketplace presenter need to tell view to close itself (i.e back) - what happens if user complete the native spend or pressing back from native spend should he return to marketplace - need to ask Product

Reply:
it's most common user flow, return to the last page you've been.